docs/guide/static-module-assets.md

```diff
@@ -0,0 +1,45 @@
+# Using static module assets
+
+While developers can place static files in the top-level `/public` folder of the project, which allows a file like `/public/images/bg.png` to be accessed via the URL `https://examplesite.com/images/bg.png`, this isn't a good solution for some situations.
+
+One issue is that the URL does not change with each new deployment, so if `bg.png` has been changed a stale version may be served by the browser's cache.
+
```

Comment:
No real need for this paragraph break since the previous paragraph and the next one are the same line of discussion and the first is only one sentence.

Reply:
Then it could simply start "Another is that..."